moveit_ros/visualization/test/test_moveit_joy.py

```diff
@@ -60,15 +57,12 @@ def tearDownClass(self):
 
     def test_updatePlanningGroup_exception(self):
         '''Test MoveitJoy.updatePlanningGroup'''
-        exception_raised = False
-        try:
-            # Passng 0 to MoveitJoy.updatePlanningGroup should raise an exception.
-            self.moveit_joy.updatePlanningGroup(0)
-        except rospy.ROSInitException:
-            exception_raised = True
-        self.assertTrue(exception_raised)
+        # Passng 0 to MoveitJoy.updatePlanningGroup should raise an exception.
+        self.assertRaises(rospy.ROSInitException, self.moveit_joy.updatePlanningGroup(0))
```

Comment:
+1 for using `assertRaises`, which seems to be the right way to check if exceptions are raised.